spec/models/railway_station_spec.rb

```diff
@@ -1,5 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe RailwayStation, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'is valid with valid attributes' do
+    expect(create(:railway_station)).to be_valid
```

Comment:
`expect(build(:railway_station)).to be_valid` - you cannot create invalid record